packages/components/src/guide/style.scss

```diff
@@ -104,5 +104,15 @@
 
 	&.components-guide__finish-button {
 		right: 0;
+
+		@media (max-width: $break-small) {
```

Comment:
Can we instead invert the style (mobile first) and use our mixins?

Reply:
I find it odd setting `display: none` only to then set `display: unset`. But happy to use the mixin if you prefer :)

Reply:
I don't like it a lot either. but you know we have to follow the gurus of mobile-first approach and stay consistent in our codebase :)

Reply:
I want you to be happy, Riad ❤️ 53974ef3c